lib/gear_application/config_getter.ex

```diff
@@ -27,6 +27,32 @@ defmodule Antikythera.GearApplication.ConfigGetter do
 
   @doc false
   def get_all_env(gear_name) do
+    case load_config_from_process_dictionary(gear_name) do
+      {:ok   , config              } -> config
+      {:error, current_gear_configs} ->
+        config = load_config_from_ets(gear_name)
+        store_config_to_process_dictionary(gear_name, current_gear_configs, config)
+        config
+    end
+  end
+
+  defp load_config_from_process_dictionary(gear_name) do
+    case Process.get(:gear_configs) do
+      nil     -> {:error, %{}}
+      configs ->
```

Comment:
`config` and `configs` can be a bit confusing. Let's use `gear_configs` for this variable.
Also let's rename `current_gear_configs` to `gear_configs` (just to make it shorter and consistent).

Reply:
fixed in 40400f1f530816cdb00b656fc408cec823d826e1